src/Microsoft.AspNetCore.Mvc.ViewFeatures/AutoValidateAntiforgeryTokenAttribute.cs

```diff
@@ -17,12 +17,14 @@ namespace Microsoft.AspNetCore.Mvc
     /// validation of antiforgery tokens by default for an application. Use
     /// <see cref="IgnoreAntiforgeryTokenAttribute"/> to suppress validation of the antiforgery token for
     /// a controller or action.
+    /// The default Order for this attribute is 1000 because it must run after any filter which does authentication 
```

Comment:
I would suggest putting this document on the order property. Also when you're working on doc comments, newlines in the source code are not significant. Use the `<para></para>` tag if you want to make paragraphs